odl/solvers/nonsmooth/primal_dual_hybrid_gradient.py

```diff
@@ -74,12 +75,20 @@ def chambolle_pock_solver(x, f, g, L, tau, sigma, niter, **kwargs):
     theta : float, optional
         Relaxation parameter, required to fulfill ``0 <= theta <= 1``.
         Default: 1
-    gamma : non-negative float, optional
+    gamma_primal : non-negative float, optional
         Acceleration parameter. If not ``None``, it overrides ``theta`` and
         causes variable relaxation parameter and step sizes to be used,
-        with ``tau`` and ``sigma`` as initial values. Requires ``G`` or
-        ``F^*`` to be uniformly convex.
+        with ``tau`` and ``sigma`` as initial values. Requires ``G`` to be
+        strongly convex. ``gamma_primal`` is the strong convexity constant of
+        ``G``. Acceleration can either be done on the primal part or the dual
```

Comment:
I don't quite understand this. It seems impossible that `gamma_primal` is both a derived property of `G` and a user choice.
Is the strong convexity constant maybe an upper bound for the choice of acceleration parameter?

Reply:
True. In practice, one wants to have gamma as large as possible. Also the use of "strong convexity constant" is often not unique. Some authors refer to it as the largest constant such that property X holds, others as any constant such that X holds.